classes/Bot/activities/workshop.js

```diff
@@ -176,7 +177,7 @@ class Workshop extends Activity {
             callback: (user, reaction, stopInteracting, console) => this.sendPoll(pollInfo.type).then(() => stopInteracting()),
         }));
 
-        localFeatures.forEach(feature => this.adminConsole.addFeature(feature));
+        localFeatures.forEach(feature => this.adminConsole.addFeature(Feature.create(feature)));
```

Comment:
can you make sure this feature var is not already an actual Feature Object?

Reply:
the `feature` var is just an element of the `localFeatures` array, and the objects in `localFeatures` have not been initialized to Features yet

Reply:
okay lovely@ Will close this comment then!